File: codewars/is a number prime.py
import math
def is_prime(num):
    if num < 2:
        return False
    if num % 2 == 0 and num != 2:
        return False
    if num % 3 == 0 and num != 3:
        return False
    for i in range(3,int(math.sqrt(num))+1):
        if num % i == 0:
            return False

    return True

File: codewars/Is a number prime.go
package kata
import "math"
func IsPrime(n int) bool {
  if(n == 0){
    return false
  }
  if(n<2){
    return false
  }
  if(n%2 == 0 && n!= 2){
    return false
  }
  if(n%3 == 0 && n!= 3){
    return false
  }
  for i:=3;i<int(math.Sqrt(float64(n)));i=i+2 {
    if(n%i == 0 ){
      return false
    }
  }
  return true
}